taxonomy-manager-rest-server/src/main/java/com/digirati/taxman/rest/server/taxonomy/autocomplete/ConceptAutocompleteSuggestionMapper.java

```diff
@@ -0,0 +1,32 @@
+package com.digirati.taxman.rest.server.taxonomy.autocomplete;
+
+import com.digirati.taxman.common.taxonomy.ConceptModel;
+
+import java.net.URI;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Maps a {@link ConceptModel} to an {@link AutocompleteSuggestion}.
+ */
+class ConceptAutocompleteSuggestionMapper {
+
+    private final String searchTerm;
+
+    ConceptAutocompleteSuggestionMapper(String searchTerm) {
+        this.searchTerm = searchTerm;
+    }
+
+    AutocompleteSuggestion map(ConceptModel conceptModel) {
+        URI uri = conceptModel.getUri();
+        Map<String, String> preferredLabel = conceptModel.getPreferredLabel();
```

Comment:
Why are we trying to figure out which language matched? We should be only matching against one language when we query the database.

Reply:
So that the frontend can determine which of the preferred labels to display. I disagree that we should only be matching against one language since the point of this is to provide concept IDs to the frontend when the user is trying to build relationships. Since relationships are based on those IDs rather than labels, the language we search for is irrelevant. Why do you think we should only be matching against one language and how do you propose that that frontend gets that language?